doc/tutorials/content/writing_new_classes.rst

```diff
@@ -1175,8 +1154,7 @@ And the *bilateral.hpp* likes:
      *
      */
 
```

Comment:
I think we could use this PR to update also the LICENSE blurb to use the new one when writing a class. What do you think @kunaltyagi ?

Reply:
Blurb in this rst file? Sure. But not in actual code files

Reply:
Yes, speaking about the doc, not actual code!

Reply:
[This one](https://github.com/PointCloudLibrary/pcl/blob/b19c99a5f45950c0e519e631d09adb0c5257596e/CONTRIBUTING.md#license)? Maybe we should also add this to the style guide

Reply:
Would it be considered a style guide? Isn't that more for code? We do have a blurb in contributing.md for reference

Reply:
Personally I would consider it as style guide ([Google Style Guide](https://google.github.io/styleguide/cppguide.html#File_Comments) also include in their guide)

But maybe updating this file is enough for new contributors, as anyone wanted to implement a new class will definitely look at this rst